setup.py

```diff
@@ -33,6 +33,8 @@
 install_requires = [
     'fmf>=0.9.2',
     'click',
+    'requests',
+    'testcloud'
```

Comment:
I think these should belong to `extra_requires`.

Reply:
fixing